### DISCUSSION TITLE
How do I enable auto-completion for powershell on Linux

### DISCUSSION THREAD

**Discussion by fuzy112** [microsoft/vcpkg#17829] [Ideas]
Hi. I am using vcpkg on **Linux** and my shell is **Powershell** 7.0.0. I want to enable auto-completion of commands, but `./vcpkg integrate powershell` did not seem to work on Linux. Is there a way to  enable
auto-completion for Powershell on Linux?

**Comment by LilyWangL** [comment]
@Chingyat Thanks for posting this issue. You can install bash-completion by command `yum -y install bash-completion` or `sudo apt-get install bash-completion`.

**Comment by fuzy112** [comment]
@LilyWangL  I am not using bash. I am using PowerShell.

**Comment by fuzy112** [comment]
@LilyWangL  You mean PowerShell can take advantage of bash-completion to complete the commands?

**Comment by JackBoosY** [comment]
vcpkg currently only supports powershell integration in windows, and will support powershell on other platforms in the future.

**Comment by JackBoosY** [comment]
Related #10662.
